### PR TITLE
ops-485 Update Archiving SOPs

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/hca-ebi-dev-team#485

https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/ops-485_update-archiving-sop/docs/SOPs/archiving_SOP.md

https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/ops-485_update-archiving-sop/docs/SOPs/update_ena_runs_SOP.md

### CODE REVIEW

File: docs/SOPs/archiving_SOP.md [hunk 1]
```diff
@@ -15,31 +15,25 @@ parent: SOPs
...
-<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>
+### Step 1: Submit for Archiving
 
-## Submit for Archiving
-
-1. Once the data files are uploaded, the submission should be listed as **Valid.**
+1. Once the data files are uploaded and graph validated, the submission should be listed as **Graph Valid.**
```
Comment (ESapenaVentura) [inline docs/SOPs/archiving_SOP.md:25]:
```suggestion
1. Once the data files are uploaded and the submission has been validated, the submission should be listed as **Graph Valid.**
```

File: docs/SOPs/archiving_SOP.md [hunk 2]
```diff
@@ -60,317 +54,40 @@ There are currently 3 environments that you can use for the archiver:
...
 
-If running parallel jobs, choose different <file_name> / <job_names> because you will have multiple file-upload-infos and multiple jobs, in this case I’ve used the name of the output bam file as the job_name and file_name.
-
-# Step 3 of 3 - Complete DSP Submission
-
-Once the files are uploaded and validated, to finish the submission, just run the following request to complete the submission:
-```
-curl -X POST https://archiver.ingest.staging.archive.data.humancellatlas.org/archiveSubmissions/<dsp_submission_uuid>/complete -H 'Api-Key: {archiver_api_key}' 
-```
+Follow Step 3 (Submit new sequencing runs) of "Update ENA runs SOP" page.
```
Comment (ESapenaVentura) [inline docs/SOPs/archiving_SOP.md:93]: Missing a hyperlink to the document

File: docs/SOPs/update_ena_runs_SOP.md [hunk 3]
```diff
@@ -62,16 +64,27 @@ This is the SOP for fixing datasets in the issue: ebi-ait/hca-ebi-wrangler-centr
...
-5. Checksum all the files.
+3. Checksum all the files.
     ``` 
     gsutil hash -hm gs://broad-dsp-monster-hca-prod-ebi-storage/prod/<project_uuid>/data/* | grep -A1 "hex" | awk -F"/" '{printf $4 $1}' | awk -F"--" '{for (i=1;i<NF;i++)print $i}' | awk -F":" '{print $1 $3}' > <md-filename>.txt
     ```
-6. Upload the files to Webin FTP upload area (could be in parallel with checksumming)
+4. Upload the files to Webin FTP upload area (could be in parallel with checksumming)
    ```
    $ cd <directory where you downloaded the files>
    $ lftp webin2.ebi.ac.uk -u <webin-user>
```
Comment (ESapenaVentura) [inline docs/SOPs/update_ena_runs_SOP.md:90]: I am missing a bit on how to get the webin user

File: docs/SOPs/update_ena_runs_SOP.md [hunk 4]
```diff
@@ -81,19 +94,10 @@ This is the SOP for fixing datasets in the issue: ebi-ait/hca-ebi-wrangler-centr
    $ mput *
    ```   
    Please refer to [ENA documentation](https://ena-docs.readthedocs.io/en/latest/update/metadata/programmatic-read.html) for more details
-7. Run the submit_10x_fastq_files.py script. The `receipt.xml` and `report.json` file should be available after running the script.
```
Comment (ESapenaVentura) [inline docs/SOPs/update_ena_runs_SOP.md:84]: hyperlink to script plz

File: docs/SOPs/update_ena_runs_SOP.md (hunk 3 above)
Comment (ESapenaVentura) [reply docs/SOPs/update_ena_runs_SOP.md:90]: (the instructions may not necessarily tell you, but it should point you out to where/who to contact)

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]

File: docs/SOPs/update_ena_runs_SOP.md (hunk 3 above)
Comment (aaclan-ebi) [reply docs/SOPs/update_ena_runs_SOP.md:90]: Good point! I've got it from Karoly before, it's the DSP webin user 😅

**Comment by aaclan-ebi** [comment]
Thanks @ESapenaVentura ! I've applied your comments! Please re-review.

**Pull request merged**